src/button.ts

```diff
@@ -145,6 +147,7 @@ export default class Button extends LitElement {
         ${this.label}
 
         <slot
+          class=${classMap({ hidden: this.variant === 'link' })}
```

Comment:
```suggestion
          class=${classMap({ link: true, hidden: this.variant === 'link' })}
```

Then in your declaration:

```
.link {
  &.hidden {
    display: none;
  }
}
```